iosxr_store_box.py

```diff
@@ -180,18 +172,19 @@ def verboseprint(*args):
 \n vagrant ssh
         """ % (sender, receiver, location, message, box_out, box_out)
 
-    verboseprint('Email is:')
-    print(email)
+    if args.verbose or test_only:
+        print('Email is:')
+        print(email)
```

Comment:
Because I have a --test option and I want to see the outputted email without sending it.

But it should be:     if args.verbose == logging.DEBUG or test_only: